ee_plugin/ee_plugin.py

```diff
@@ -112,36 +112,54 @@ def initGui(self):
             parent=self.iface.mainWindow(),
             triggered=self._run_cmd_set_cloud_project,
         )
+        add_fc_button = QtWidgets.QAction(
+            text=self.tr("Add Feature Collection"),
+            parent=self.iface.mainWindow(),
+            triggered=lambda: forms.add_feature_collection_form(self.iface),
+        )
 
-        # Build plugin menu
+        # Initialize plugin menu
         plugin_menu = cast(QtWidgets.QMenu, self.iface.pluginMenu())
-        ee_menu = plugin_menu.addMenu(
+        self.menu = plugin_menu.addMenu(
             icon("earth-engine.svg"),
             self.tr("&Google Earth Engine"),
         )
-        self.menu = ee_menu
-        ee_menu.addAction(ee_user_guide_action)
-        ee_menu.addSeparator()
-        ee_menu.addAction(sign_in_action)
-        ee_menu.addAction(self.set_cloud_project_action)
-
-        # Build toolbar
-        toolButton = QtWidgets.QToolButton()
-        toolButton.setToolButtonStyle(
+
+        # Initialize toolbar menu
+        self.toolButton = QtWidgets.QToolButton()
+        self.toolButton.setToolButtonStyle(
             Qt.ToolButtonStyle.ToolButtonIconOnly
             # Qt.ToolButtonStyle.ToolButtonTextBesideIcon
         )
-        toolButton.setPopupMode(
-            QtWidgets.QToolButton.ToolButtonPopupMode.MenuButtonPopup
+        self.toolButton.setPopupMode(
+            # QtWidgets.QToolButton.ToolButtonPopupMode.DelayedPopup  # Button is only for triggering action
+            QtWidgets.QToolButton.ToolButtonPopupMode.InstantPopup  # Button is only for opening dropdown menu
+            # QtWidgets.QToolButton.ToolButtonPopupMode.MenuButtonPopup  # Button is split into action and dropdown menu
+        )
+        self.toolButton.setMenu(QtWidgets.QMenu())
+        self.toolButton.setDefaultAction(
+            QtWidgets.QAction(
+                icon=icon("earth-engine.svg"),
+                text=f'<strong>{self.tr("Google Earth Engine")}</strong>',
+                parent=self.iface.mainWindow(),
+            )
         )
-        toolButton.setMenu(QtWidgets.QMenu())
-        toolButton.setDefaultAction(ee_user_guide_action)
-        toolButton.menu().addAction(ee_user_guide_action)
-        toolButton.menu().addSeparator()
-        toolButton.menu().addAction(sign_in_action)
-        toolButton.menu().addAction(self.set_cloud_project_action)
-        self.iface.pluginToolBar().addWidget(toolButton)
-        self.toolButton = toolButton
+        self.iface.pluginToolBar().addWidget(self.toolButton)
+
+        # Add actions to the menu
+        for action in [
+            ee_user_guide_action,
+            None,
```

Comment:
Is none used for some whitespace or something? Wondering if we should extract this to another function that potentially takes a list of actions, and figures out the formatting. With the sub-menu logic, we could also maybe use a dictionary that maps to the sub-menu/level desired? Again, we can refactor after this PR.

Reply:
Yeah, this is something that I was unsure about.  If you look below, you'll see:

```py
                if action:
                    menu.addAction(action)
                else:
                    menu.addSeparator()
```

So basically, `None` represents a separator.  That feels like an awkward convention but wasn't sure what would be better.  I also am unsure that we want to keep the plugin menu and toolbar menu completely in sync as we are doing now.

Reply:
In https://github.com/gee-community/qgis-earthengine-plugin/pull/231/commits/6036c7c1f7d8a327f3318062b6be258f90351ec9, I abstracted how the menu is setup.